app/packages/app/src/pages/datasets/DatasetPage.tsx

```diff
@@ -122,8 +121,7 @@ const DatasetPage: Route<DatasetPageQuery> = ({ prepared }) => {
           </datasetQueryContext.Provider>
         )}
       </div>
-      <Snackbar />
```

Comment:
Is the removal of `Snackbar` intentional?

Reply:
Unintentional 🤦 Thanks!